images/ansible-operator/Pipfile

```diff
@@ -4,13 +4,16 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-ansible-runner = "~=2.3.3"
+ansible-core = "==2.17.4"
+ansible-runner = "~=2.4.0"
 ansible-runner-http = "~=1.0.0"
-ansible-core = "~=2.15.9"
-urllib3 = "~=1.26.17"
+urllib3 = "~=1.26.2"
```

Comment:
Are we downgrading the `urllib3` version?

Reply:
The Pipfile.lock is generated with `urlib3` pinned to `1.26.20`: https://github.com/operator-framework/ansible-operator-plugins/pull/92/files#diff-82452578a3e817b3b4dc5c154471041589c4b535216c59dbd8982bc18e2f8d32R880